corehq/apps/linked_domain/const.py

```diff
@@ -20,6 +21,7 @@
     (MODEL_FIXTURE, ugettext_lazy('Lookup Table')),
     (MODEL_CASE_SEARCH, ugettext_lazy('Case Search Settings')),
     (MODEL_REPORT, ugettext_lazy('Report')),
+    (MODEL_DATA_DICTIONARY, ugettext_lazy('Data Dictionary')),
```

Comment:
Can you run `makemigrations` so this gets added as an option to the `DomainLinkHistory` model?

Reply:
Sure, migration has been added.